spec/controllers/task_managements_controller_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "rails_helper"
+
+RSpec.describe TaskManagementsController, type: :controller do
+  before(:all) do
+    @user = create(:user)
+    ironing = create(:task_management, task_desc: "Iron these clothes for me")
```

Comment:
Why not use Faker gem to generate the random strings? https://github.com/stympy/faker